app/Http/Requests/AuthRequest/SignupRequest.php

```diff
@@ -62,6 +62,7 @@ public function attributes(): array
         return [
             'first_name'        =>      'First Name',
             'last_name'         =>      'Last Name',
+            'email'             =>      'Email Address',
             'password'          =>      'Password',
             'confirm_password'  =>      'Confirm password field',
             // 'current_time'      =>      'Current time',
```

Comment:
Confirm this from Bilal if he's going to send time from Front-end or we would be getting it on Backend.